api/generic-cti/newcall-event.rst

```diff
@@ -0,0 +1,269 @@
+New call
+========
+
+.. include:: /api/generic-cti/cti-endpoint-limitations.include.rst
+
+.. list-table:: Available attributes and sample data for newCall events
+   :widths: 20, 40, 40
+   :header-rows: 1
+
+   * - Attribute
+     - Possible value
+     - Description
+   * - ``event``
+     - ``newCall``
+     - Tell Zammad there's a new call
+   * - ``from``
+     - e.g. ``493055571600``, ``02214710334``, ``anonymous``
+     - Number that initiated the call
+   * - ``to``
+     - e.g. ``49221470334``, ``03023125771``
+     - Number that is being called
+   * - ``direction``
+     - ``in`` or ``out``
+     - The call direction - if your agent initiates a call this will be ``out``
+   * - ``callId``
+     - e.g. ``53ba82e2bd6d12d9fb2d3838f0cfb070``, ``5fb9532f40da834a``,
+       ``123456789``
+     - An ID that is unique for the call. Zammad will use this ID to identify
+       an existing call with following actions
+       (e.g. like answering or hanging up)
+
+       *This ID must be unique per call session*.
+   * - ``user``
+     - e.g. ``[John Doe]``, ``[Alice, Bob]``
```

Comment:
I think this should only be ``"John Doe"``, ``"Alice","Bob"``

The `[]` are only used in html form params to for the params als array.

Reply:
It highly depends on how the call is being made.
The old documentation didn't mention it at all and when a user wanted to contribute in https://github.com/zammad/zammad-documentation/pull/159 we noticed that the wording on how to describe this situation is quite hard actually.

I currently am trying to figure out how to note that wording wise.

Reply:
How about ...

```
The user(s) real name involved. You may have to provide array style (``[]``) params depending on the call method you choose.

| If the direction is ``out``, this is the name of the calling person(s).
| If the direction is ``in``, this is the name of the called person(s).

This value is optional.
```

Together with removed bracked for single values?

(instead of:)
```
The user(s) real name involved.

| If the direction is ``out``, this is the name of the calling person(s).
| If the direction is ``in``, this is the name of the called person(s).

This value is optional.
```

Reply:
I like our proposal a lot!